src/lib/markbind/src/parser.js

```diff
@@ -296,8 +327,28 @@ Parser.prototype._preprocess = function (node, context, config) {
       const segmentSrc = cheerio.parseHTML(fileContent, true);
       const $ = cheerio.load(segmentSrc);
       const hashContent = $(includeSrc.hash).html();
-      let actualContent = (hashContent && isTrim) ? hashContent.trim() : hashContent;
 
+      if (isImport) {
+        const variableContent = $(`variable[name=${includeSrc.hash.substring(1)}]`).html();
+        if (!variableContent) {
+          // eslint-disable-next-line no-console
+          console.warn(`Missing import variable ${includeSrc.pathname} in ${element.attribs[ATTRIB_CWF]}.`);
+          return createEmptyNode();
+        }
+        if (!context.importedVariables) {
+          // eslint-disable-next-line no-param-reassign
+          context.importedVariables = {};
+        }
+        if (!context.importedVariables[includeSrc.pathname]) {
+          // eslint-disable-next-line no-param-reassign
+          context.importedVariables[includeSrc.pathname] = [];
+        }
+        // eslint-disable-next-line no-param-reassign
```

Comment:
No need to disable eslint rule for this one.

Reply:
Nevermind, we can let someone else clean this up as part of first-timer issue, let's merge it if there's no major issues with the code.

Reply:
Ahh sure